src/helpers/helpers.math.js

```diff
@@ -38,3 +42,42 @@ export const log10 = Math.log10 || function(x) {
 
 	return isPowerOf10 ? powerOf10 : exponent;
 };
+
+export function pointInLine(p1, p2, t) {
```

Comment:
Do we want all these new methods to be public or should we make some of them private?

Reply:
Not sure. These are quite generic. I did not have a look if these would be available in a separate lib we could import from.